lib/tasks/eu_exit_business_finder.rake

```diff
@@ -0,0 +1,21 @@
+namespace :eu_exit_business_finder do
+  desc <<-DESC
+    Adds a link to the EU Exit Business Readiness Finder to all content items which are tagged to this finder
+  DESC
+  task add_related_link_to_tagged_content: [:environment] do
+    facet_group_content_id = "52435175-82ed-4a04-adef-74c0199d0f46"
+    finder_content_id = "42ce66de-04f3-4192-bf31-8394538e0734"
```

Comment:
I don't expect the finder content item is going to change, but it you wanted to, you could use `Facets::FinderService::LINKED_FINDER_CONTENT_ID` here instead. I don't think the facet group content id is stored anywhere apart from the config file though

Reply:
Have you considered adding a test for this task?

Reply:
I hardcoded the content IDs since the task will probably only be run once, then removed.  So it didn't see to give any benefit by using the `LINKED_FINDER_CONTENT_ID` value here.

I'll look into adding a test 😬.

Reply:
I've had second thoughts around adding a test for the rake task.  I'm stubbing all the requests (before and after the related link gets added), so it's not actually testing anything, other than the correct publishing-api endpoints are being requested.

This code could be moved into self contained methods, then each tested with a unit test, but this feels against the ethos of optimising code for deletion since this task will be removed after it has been run in production.